packages/components/Organisms/Accordion/Accordion.vue

```diff
@@ -88,23 +80,56 @@ export default {
         let strIndex = index.toString()
         for (let item in this.itemOpen) {
           if (item !== strIndex) {
+            this.collapseContent(item)
             Vue.set(this.itemOpen, item, false)
           }
         }
       }
+
+      this.toggleContent(index)
       Vue.set(this.itemOpen, index, !this.itemOpen[index])
     },
-    start (el) {
-      el.style.height = el.scrollHeight + 'px'
-    },
-    end (el) {
-      el.style.height = ''
-    },
     accordionIsOpen: function (index) {
       return (this.itemOpen[index] === undefined) ? false : this.itemOpen[index]
     },
     accordionId (index) {
       return `rpl-accordion-${this.getIdFromLocalRegistry(index)}`
+    },
+    expandContent (index) {
+      const ref = this.accordionId(index)
+      const section = this.$refs[ref]
+      if (section[0]) {
+        // Set fixed height for transition
+        section[0].style.height = section[0].scrollHeight + 'px'
+        section[0].style.visibility = 'visible'
+        setTimeout(function () {
+          // Remove the fixed height after transition so it can be responsive
+          section[0].style.height = 'auto'
+        }, 500)
+      } else {
+        throw new Error('Something is wrong while getting the height of the referred content')
+      }
+    },
+    collapseContent (index) {
+      const ref = this.accordionId(index)
+      const section = this.$refs[ref]
+      if (section[0]) {
+        // Set fixed height for transition
+        section[0].style.height = section[0].scrollHeight + 'px'
+        setTimeout(function () {
+          section[0].style.height = ''
+          section[0].style.visibility = ''
+        }, 1)
```

Comment:
Interesting, this is like .001 seconds :P Do we still need to set timeout here?

Reply:
This is an ugly work around to make the transition work. The reason is I need to have fixed height fist, then change it to 0. Without this 1 millisecond delay, the transition just not work for me. It's welcome if you have better way, but I can't find good solution based on the time for this.

Reply:
Not really, just find it interesting how it works with that amount of millisecond waiting time.